src/app/edit-create-study-set/edit-create-study-set.component.scss

```diff
@@ -0,0 +1,106 @@
+@use '@angular/material' as mat;
+@use '@angular/material-experimental' as matx;
+
+
+@mixin theme($theme) {
+
+  .upper-components{
+    padding-left: 6%;
+    padding-right: 6%;
+    line-height:0em;
+  }
+
+  .first-row{
+    display: flex;
+    align-items: center;
+    vertical-align: baseline;
+  }
+
+  .page-title{
+    font: mat.get-theme-typography($theme, headline-large);
+    line-height:0em;
+  }
```

Comment:
make this the global style for `h1` in the `src/styles.scss` file. In `styles.scss` add:
```css
h1 {
  font: mat.get-theme-typography(theme.$ponder-light-theme, headline-large);
  line-height:0em;
}
```